documentation/modules/proc-upgrading-the-cluster-operator.adoc

```diff
@@ -5,11 +5,12 @@
 [id='proc-upgrading-the-co-{context}']
 = Upgrading the Cluster Operator to a later version
 
-This procedure describes how to upgrade a Cluster Operator deployment to a later version.
+This procedure describes how to upgrade a Cluster Operator deployment to a later version
```

Comment:
Out of curiosity ... why should the `.` not be here?

Reply:
It should. And I've returned it.